components/common/Paginaion.tsx

```diff
@@ -0,0 +1,69 @@
+import { useEffect, useState } from "react";
+import styles from "@/styles/Pagination.module.scss";
+
+interface PaginationProps {
+  currentPage: number;
+  totalPage: number;
+  onPageChange: (pageNumber: number) => void;
+}
+
+export default function Pagination({
+  currentPage,
+  totalPage,
+  onPageChange,
+}: PaginationProps) {
+  const [pageList, setPageList] = useState<number[]>([]);
+
+  const createPageList = (totalPage: number): number[] => {
+    return Array.from({ length: totalPage }, (v, i) => i + 1);
+  };
+
+  useEffect(() => {
+    if (totalPage === 1) return () => {};
```

Comment:
@ssseeo0 

js semantic syntax를 지켜주세요!

```suggestion
    if (totalPage === 1) return () => {};
    
    // 이 코드는 다음고 ㅏ같이 판단될 수도 있어보입니다
    
    if (totalPage === 1) {
      return () => {}
    }
```